odoo/models.py

```diff
@@ -5921,6 +5888,7 @@ def refresh(self):
             .. deprecated:: 8.0
                 The record cache is automatically invalidated.
         """
+        _logger.warning('Refresh is deprecated method, use invalidate_cache instead')
```

Comment:
You don't want this thing to fill in you log files if you ever use it on a production database.
I suggest to use `warnings.warn` instead.